grouper/permissions.py

```diff
@@ -252,20 +229,20 @@ def get_groups_by_permission(session, permission):
     )
 
 
-def filter_grantable_permissions(session, grants, all_permissions=None):
-    """For a given set of PERMISSION_GRANT permissions, return all enabled
-    permissions that are grantable.
+def filter_grantable_permissions(
+    session: Session, grants: List[Any], all_permissions: Optional[Dict[str, Permission]] = None
```

Comment:
Ah, the `Any` because they're raw sqlalchemy objects?

Reply:
Yeah, and not even full model objects, but tuples. And SQLAlchemy itself doesn't appear to be typed, so it's not clear what to use for a type other than `Any`. This will go away once we have a repository layer wrapping all of the SQLAlchemy code.